src/app/flows/fines/fines-acc/fines-acc-debtor-add-amend/constants/fines-acc-debtor-add-amend-field-errors.constant.ts

```diff
@@ -0,0 +1,278 @@
+import { IFinesAccDebtorAddAmendFieldErrors } from '../interfaces/fines-acc-debtor-add-amend-field-errors.interface';
+
+export const FINES_ACC_DEBTOR_ADD_AMEND_FIELD_ERRORS: IFinesAccDebtorAddAmendFieldErrors = {
+  facc_debtor_add_amend_title: {
+    required: {
+      message: 'Select a title',
+      priority: 1,
+    },
+  },
+  facc_debtor_add_amend_forenames: {
+    required: {
+      message: `Enter defendant's first name(s)`,
+      priority: 1,
+    },
+    maxlength: {
+      message: `Defendant's first name(s) must be 20 characters or fewer`,
+      priority: 2,
+    },
+    lettersWithSpacesPattern: {
+      message: `Defendant's first name(s) must only contain letters`,
+      priority: 2,
+    },
+  },
+  facc_debtor_add_amend_surname: {
+    required: {
+      message: `Enter defendant's last name`,
+      priority: 1,
+    },
+    maxlength: {
+      message: `Defendant's last name must be 30 characters or fewer`,
+      priority: 2,
+    },
+    lettersWithSpacesPattern: {
+      message: `Defendant's last name must only contain letters`,
+      priority: 2,
+    },
```

Comment:
Incorrect priority number: 

```suggestion
    lettersWithSpacesPattern: {
      message: `Defendant's last name must only contain letters`,
      priority: 3,
    },
```